src/components/VizbeeMiniCastBar/index.js

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { View, Image, TouchableOpacity, Text, Appearance } from "react-native";
+import {
+  useMiniCastBarMediaStatusListenerHook,
+  useMiniCastBarSessionListenerHook,
+  useMiniCastBarSessionInfoHook,
+  useMiniCastBarActionsHook,
+  useMiniCastBarStyleHook,
+} from "./hooks";
+import {
+  getTitleFromVideoInfo,
+  getSubTitleFromVideoInfo,
+  getApperanceColor,
+} from "./helpers";
+import PropTypes from "prop-types";
+import { Images } from "./constants";
+
+const VizbeeMiniCastBar = ({
+  thumbnailImageSourceDefault,
+  backgroundLayerStyle,
+  progressViewStyle,
+  titleStyle,
+  subTitleStyle,
+  buttonStyle,
+  disableButtonStyle,
+  playActionButtonType,
+  shouldDisablePlayPauseButtonForLive,
+  buttonImgPlay,
+  buttonImgPause,
+  buttonImgStop,
+}) => {
+  const isDark = Appearance.getColorScheme() === "dark";
+
+  const defaultApperance = getApperanceColor(isDark);
```

Comment:
is this typo? `defaultApperance` --> defaultAppearance` and `getApperanceColor` --> `getAppearanceColor`

Reply:
Done